test/test_prepared.py

```diff
@@ -0,0 +1,14 @@
+import psycopg
+import pytest
+
+def test_prepared_statement(bouncer):
+    bouncer.admin(f"set pool_mode=transaction")
```

Comment:
I think this should make the test pass again

```suggestion
    bouncer.admin(f"set pool_mode=transaction")
    bouncer.admin(f"set prepared_statement_cache_size=100")
```